File: local.c
#include <stdio.h>
#include <fcntl.h>
#include <unistd.h>

int main(void)
{
    char buf[200] = {0};
    int fd;
    ssize_t status;

    fd = open("test_file.txt", O_RDWR);
    status = read(fd, buf, 5);
    printf("%ld %s\n", status, buf);
    lseek(fd, 0, SEEK_SET);
    status = read(fd, buf + 5, 6);
    printf("%ld %s\n", status, buf);
    status = write(fd, "word", 4);
    close(fd);

    return 0;
}

File: local.py
import os

fd = os.open("test_file.txt", os.O_RDWR)
buf = os.read(fd, 5)
print(str(len(buf)) + " " + buf.decode())
os.lseek(fd, 0, os.SEEK_SET)
buf2 = os.read(fd, 6)
print(str(len(buf2)) + " " + buf.decode() + buf2.decode())
status = os.write(fd, "word".encode())
os.close(fd)
